wdl/Vapor.wdl

```diff
@@ -11,6 +11,8 @@ workflow Vapor {
     File bed_file
     String sample_id
 
+    Boolean save_plots  # Control whether plots are produced in final output
```

Comment:
```suggestion
    Boolean save_plots  # Control whether plots are final output
```
This is really nit-picky, but as we've discussed, the plots are produced either way - this is a question of whether they are designated as final workflow outputs, which can translate to whether they get saved vs. deleted

Reply:
Understood, fix was made. I guess it can translate either way but I see how that makes it clearer. Thank you!